### PR TITLE
Mile Stone : JavaScript Classes

### PR DESCRIPTION
**Pull request by ahmed025566**
In this milestone you can see that I have made the following: 

- I have used classes instead of functions.
- I have added some cool and simple style to the elements of the page.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,74 +1,89 @@
...
   }
 });
-/* eslint eqeqeq: "off" */
\ No newline at end of file
+window.addEventListener('DOMContentLoaded', () => {
+  Libraray.displayData();
+  // remove from the dom
+  Libraray.removebook();
+});
+/* eslint max-classes-per-file: "off" */
```
Comment (Ghiftee) [inline index.js]: - [ ] Let's try not to disable linters as they help us write cleaner code. You can consider splitting the code in this file into different files

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @ahmed025566 and @everinurmind  👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ App runs smoothly
✔️ README is professional
✔️ No linter errors

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly ensure your MIT license is not broken as this adds to the professionalism of your README

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: library.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+const booksDiv = document.querySelector('.books');
+const title = document.querySelector('.title');
+const author = document.querySelector('.author');
+const add = document.querySelector('.add');
+/* eslint no-undef: "off" */
```
Comment (mikethreels) [inline library.js]: - [ ] as the previous CR already mentioned Let's try not to disable linters as they help us write cleaner code. You can consider splitting the code in this file into different files.

File: README.md [hunk 3]
```diff
@@ -52,10 +52,10 @@
...
 > In this project you can see that it has the following
 
-- I have created a function that add a new book and it is trigered when the user clicks on add button.
-- Same Function add the book to local storage and the page.
-- There is a function that trigred when the user reload the page and it is populatingthe data from local stotrgae to the array of books.
-- I have created a function to delete the corsponding book .
+- I have created a class that add a new book and it is trigered when the user clicks on add button.
+- Same class add the book to local storage and the page.
+- There is a method that is trigred when the user reload the page and it is populating the data from local     stotrgae to the array of books.
+- I have created a method to delete the corsponding book .
```
Comment (mikethreels) [inline README.md:59]: - [ ] the MIT license link should also be fixed as mentioned by the previous CR

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- linters are passing

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**